doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-cloudformation-template-service-provisioning.adoc

```diff
@@ -0,0 +1,90 @@
+
+[[example-adding-orchestration-template]]
+==== Creating an Orchestration Template
+
+Complete the following procedure to add new orchestration templates. 
+
+. Navigate to menu:Services[Catalogs] and select *Orchestration Templates* in the accordion menu.
+. Click image:1847.png[Configuration] *Configuration*, then click image:1848.png[Green_Plus_Sign] *Create a new Orchestration Template*. 
+. Enter a *Name* and *Description* for your template. 
+. Select *Amazon CloudFormation* from the *Template Type* list.
+. Select *Draft* to create a draft template.
+. Add your template in the area below for the selected *Template Type*. You can author your own stack template, or can add it here from other sources.
+. Click *Add*.
+
+You can add this template as a catalog item to a service catalog. Stacks can then be created from templates and launched from the service catalog.
+
+[[example-adding-service-dialog-from-orchestration-template]]
+=== Creating a Service Dialog from an Orchestration Template
+
+Complete the following procedure to create a service dialog based on the input parameters defined in the orchestration template.
+
+. Navigate to menu:Services[Catalogs] and click *Orchestration Templates* in the accordion menu.
+. Expand *All Orchestration Templates*, then click the orchestration template you created using the above procedure, that you want to create a service dialog from.
+. Click image:1847.png[Configuration]*Configuration*, then click image:1848.png[Green_Plus_Sign]*Create Service Dialog from Orchestration Template*.
+. Enter a name for the service dialog in *Service Dialog Name*.
+. Click *Save*.
+
+
+[[example-adding-a-new-catalog]]
+=== Creating a Catalog
+
+Complete the following procedure to create a new catalog.
+
+. Navigate to menu:Services[Catalogs] and select *Catalogs* in the accordion menu.
+. Click image:1847.png[Configuration] *Configuration*, then click image:1848.png[Green_Plus_Sign] *Add a New Catalog*. 
+. In *Basic Info*, add *Name* and *Description* for the new catalog.
+. You can assign catalog items in *Assign Catalog Item*.
+. Click *Add*.
+
+
+[[example-adding-orchestration-catalog-item]]
+=== Creating an Orchestration Catalog Item
+
+Complete the following procedure to create a new orchestration catalog item.
+
+. Navigate to menu:Services[Catalogs] and select *Catalog Items* in the accordion menu.
+. Click image:1847.png[Configuration] *Configuration*, then click image:1848.png[Green_Plus_Sign] *Add a New Catalog Item*. 
+. Select *Orchestration* from the *Catalog Item Type* list.
+. Enter the basic details in the *Basic Info*:
+.. Enter a *Name* and *Description* for the new service catalog item. 
+.. Select *Display in Catalog* box.
+.. Select the appropriate catalog from the *Catalog* list.
+.. Select the appropriate dialog from the *Dialog* list.
+.. Select the orchestration template from the *Orchestration Template* list.
```

Comment:
I'm not sure exactly why, but at this step I cannot add a template (none appear in the drop down) and there's no Provider option for me. I've followed all steps so far.

Reply:
Fixed.